umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -181,3 +189,18 @@
       (assoc :MetadataSpecification {:URL "https://cdn.earthdata.nasa.gov/umm/granule/v1.6.1"
                                      :Name "UMM-G"
                                      :Version "1.6.1"})))
+
+(defmethod interface/migrate-umm-version [:granule "1.6.2" "1.6.1"]
+  [context g & _]
+  (-> g
+      (assoc :MetadataSpecification {:URL "https://cdn.earthdata.nasa.gov/umm/granule/v1.6.1"
+                                     :Name "UMM-G"
+                                     :Version "1.6.1"})
+      (util/update-in-each [:RelatedUrls] v1-6-2-related-url-type->1-6-1)))
+
+(defmethod interface/migrate-umm-version [:granule "1.6.1" "1.6.2"]
```

Comment:
Do you need to migrate existing s3 RelatedURLs that are `GET DATA` type in v1.6.1 to the new type in v1.6.2?

Reply:
No according to Doug Newman